SlimApp/View/ClientDetailsView.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  ClientDetailsView.swift
+//  SlimApp
+//
+//  Created by Antonio Palomba on 11/01/23.
+//
+
+import SwiftUI
+
+struct ClientDetailsView: View {
+    var body: some View {
+        VStack {
+            
+            Circle()
+                .foregroundColor(.gray)
+                .frame(width: 200, alignment: .center)
+                .overlay {
+                    Image(systemName: "person.fill")
+                        .foregroundColor(.white)
+                        .font(.custom("sf-arabic", fixedSize: 150))
+                }
+            
+            List {
```

Comment:
I think it is better to use Form in such case or customize List. The reason is that i don't think that we have a need to hide our sections under the title

Reply:
I analysed the problem, and decided to keep the functionalities that List gives us while hiding the chevron button for hiding/showing sections with `.listStyle(.insetGrouped)`